src/App.jsx

```diff
@@ -20,6 +20,7 @@ import Hidden from 'material-ui/Hidden';
 import IconButton from '@material-ui/core/IconButton';
 import MenuIcon from '@material-ui/icons/Menu';
 
+
```

Comment:
If we are putting local imports at the end, then we don't necessarily need to have added newlines here.